.github/workflows/deploy-and-test.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
     needs: [deploy]
     uses: ./.github/workflows/starknet-rs-tests.yml
     secrets:
-      STARKNET_RPC: ${{ secrets.RPC_URL }}/v0_7
+      STARKNET_RPC: ${{ secrets.RPC_URL }}/v0_8
```

Comment:
Maybe not this PR, but I think we can run against 2 versions for v0_7 and v0_8 similar to starknet-go tests